docs/prover/checking/sanity.md

```diff
@@ -39,15 +38,21 @@ of the failure:
 The remainder of this document describes these checks in detail.
 
 (sanity-reachability)=
-Reachability checks
--------------------
-
-**Reachability** checks that even when ignoring all the user-provided
-assertions, the end of the rule is reachable. This check ensures that that
-the combination of `require` statements does not rule out all possible
-counterexamples.
+Vacuity checks
+--------------
+
+The **rule vacuity** sanity rule checks that even when ignoring all the
```

Comment:
I think I understand it now.  "rule vacuity" is the name, because it shows vacuity of a rule.  But the sentence reads strange if you don't know this.

I have no good suggestion; Calling it "rule-vacuity" may face the same problem.

Reply:
I replaced it (here and elsewhere) with "the _ sanity check ensures..."